LICENSE-APACHE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2023  ZKID Labs AG
+Copyright (C) 2023  Zero Block Global Foundation
```

Comment:
According to 4.(C) of that Apache license you should include all authors into the license copyright.